.env.sample

```diff
@@ -3,6 +3,10 @@ SERVICE_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyb2xlIjoic2VydmljZV9yb2xlIiw
 TENANT_ID=bjhaohmqunupljrqypxz
 REGION=region-of-your-s3-bucket
 GLOBAL_S3_BUCKET=name-of-your-s3-bucket
+GLOBAL_OSS_BUCKET=
```

Comment:
You don't need this variable, we always use `GLOBAL_S3_BUCKET` maybe we can rename it to `GLOBAL_BUCKET` to make it more generic in a future PR